lib/Launcher.js

```diff
@@ -82,6 +82,25 @@ class Launcher {
       chromeArguments.push('--auto-open-devtools-for-tabs');
       options.headless = false;
     }
+    if (options.deterministic) {
+      if (options.deterministic.date)
+        chromeArguments.push('--initial-virtual-time=' + options.deterministic.date.getTime() / 1000);
+      options.deterministic.beginFrameControl = options.deterministic.beginFrameControl || true;
+      if (options.deterministic.beginFrameControl) {
```

Comment:
Can we get away with just two flags:
- `determenistic` that will assume `beginFrameControl` and determenistic fetch
- `determenisticTime` that will set the initial virtual time and will work only together with `determenistic` option